### DISCUSSION TITLE
What means of `mixed text` option in formula input?

### DISCUSSION THREAD

**Discussion by Deeperand** [excalidraw/excalidraw#6378] [Q&A]
Hi, I had a question about the usage of  https://math.preview.excalidraw.com/ . I noticed in math-input pane there is an option `mixed text`. But after I click the option, formula won't be rendered any more even I close the formula with a pair of dollar symbol `$`. Do anyone know what can do with this `mixed text` option?

[the remark this answers is not in the document]
**Reply by DanielJGeiger** [reply]
Hi @Deeperand.  In mixed-text mode, you should put LaTeX between the delimiters `\(` and `\)`.

(Some time ago, I switched the delimiters from `$$` and `$$` to `\(` and `\)` to match standard LaTeX.)

[the remark this answers is not in the document]
**Reply by Deeperand** [reply]
@DanielJGeiger 

Thanks for your reply! I've tried using `\(` and `\)` as delimiters, but it seems doesn't work (I also tried preview in `View mode`, it also doesn't work), if anything I missed?

<img width="1168" alt="Screen Shot 2023-03-20 at 16 34 33" src="https://user-images.githubusercontent.com/42906615/226287457-7f3a0051-a3eb-4cfb-b631-ed237edd2a75.png">

**Comment by DanielJGeiger** [comment]
HI @Deeperand, try right-clicking the math button in the toolbar.  Does the context menu show "Standard input" (LaTeX) selected?  Or does it show "Simplified input" (AsciiMath) selected?

**Comment by julien-blanchon** [comment]
`\(` and `\)` work well for me
